lib/async-emitter.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+class AsyncEmitter {
+  constructor() {
+    this.events = new Map();
+  }
+
+  // Get or create event
+  //   name <string> event name
+  // Returns: { on: Set { <Function> }, once: Set{ <Function> } }
```

Comment:
This isn't rendered correctly `_Returns:_ { on [`<Function>`][function], once [`<Function>`][function] }`
I don't think we have syntax for types of set, though I guess `Returns: { on: <Set>, once: <Set> }` should be fine.